composer.json

```diff
@@ -21,6 +21,14 @@
     "require-dev": {
         "behat/symfony2-extension": "^2.1",
         "internations/http-mock": "^0.12",
-        "squizlabs/php_codesniffer": "~3.1.1"
-    }
+        "squizlabs/php_codesniffer": "~3.1.1",
+        "ubirak/rest-api-behat-extension": "^7.0",
+        "swaggest/json-diff": "dev-master"
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:atomatis/json-diff.git"
```

Comment:
Why do we need to use your fork ?

Reply:
The component don't have simple tools for get diff value between original and new Json string for a specific path.
Without the fork, i need to add complex parsing method in context, losing most of interest to use this component.